docs/concepts/html/lists.md

```diff
@@ -4,7 +4,8 @@ title: Lists
 
 ## Fragments
 
-The `html!` macro always requires a single root node. In order to get around this restriction, it's valid to wrap content in empty tags:
+The `html!` macro always requires a single root node. In order to get around this restriction, you
+can use an "empty tag" (these are also called "fragments" in the React world).
```

Comment:
```suggestion
can use an "empty tag" (these are also called "fragments").
```
"fragments" is used in the html macro error message so I think this is in our world too :)